app/locales/mr/index.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@alpita-masurkar should we just throw away all old translations and start from scratch?  Thoughts?

Reply:
I think it's probably pretty late for @alpita-masurkar ... but I did discuss this in a chat with her earlier today ...

> What do you think about asking translators to start again.  Do you see any issue with that?
> 
> Alpita Masurkar  12:49 PM
> not really,
> 12:49
> this was one of the first things I had discussed with them
> 12:49
> we have some translators who are familiar with github and directly create their prs
> 12:50
> the others had already shown their willingness to help out and are aware that this is an actively developing app, so the content will keep getting updated

Reply:
Thanks, we'll reset for now. Any old strings are here in the history.